Oxford Reference.js

```diff
@@ -0,0 +1,226 @@
+{
+	"translatorID": "62415874-b53c-4afd-86e8-814e18a986f6",
+	"label": "Oxford Reference",
+	"creator": "Sonali Gupta",
+	"target": "http://www.oxfordreference.com/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-06-15 08:51:48"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Sonali Gupta
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.indexOf("/search") != -1)
+		return "multiple";
+	else
+	{
+		var body = doc.getElementsByTagName("body")[0];
+		if ((body.className).indexOf('dctype-oxencycl-entry') != -1) {
+   			return "bookSection";
+		}
+		else
+   			return "book";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//span[@class="titlePart"]/a');
+	var rowsExtendedTitle = ZU.xpath(doc, '//span[@class="title"]');
```

Comment:
I think it would then be better to try something like:
```js
var rows = ZU.xpath(doc, '//span[@class="titlePart"]');
...
   var href = ZU.xpathText(rows[i], './a/@href');
   var title = ZU.trimInternal(rows[i].textContent);
```
(Note: this is untested code.)